src/main/resources/db/migration/V1_98__create_draft_referral_update_referral_constraints.sql

```diff
@@ -0,0 +1,42 @@
+INSERT INTO metadata (table_name, column_name, sensitive, domain_data)
+  select 'draft_referral',column_name,sensitive,domain_data from metadata  where table_name='referral';
+
+CREATE TABLE draft_referral AS SELECT * FROM referral;
```

Comment:
Can we find out what locking mechanism would this use during the migration?

- If it locks `referral` for inserts, we cannot deploy this during working hours
- If it does not lock, we have a potential issue of missing data and we need to do a follow-up copy after the deploy

Reply:
This query uses ACCESS SHARE LOCK and being cautious as it is a big change, it would be good during quiet hours